hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -985,14 +985,53 @@
     { name: "PART.DATA_REG.INTEGRITY",
       desc: "All partition buffer registers are protected with ECC on 64bit blocks."
     }
+    { name: "PART.DATA_REG.BKGN_CHK",
+      desc: "The digest of buffered partitions is recomputed and checked at pseudorandom intervals in the background."
+    }
+    { name: "PART.MEM.REGREN"
+      desc: "Unbuffered ('software') partitions can be read-locked via a CSR until the next system reset."
+    }
+    { name: "PART.MEM.SW_UNREADABLE"
+      desc: "Secret buffered partitions become unreadable to software, once they are locked via the digest."
```

Comment:
Grammar nit: You don't actually need the comma before "once" here and it looks a little odd.

Reply:
ah ok, thanks!